content/articles/oop-in-java-abstraction-and-encapsulation/index.md

```diff
@@ -1,350 +1,482 @@
-#**The concept of OOP In Java Getting Started with ABSTRACTION and ENCAPSULATION**
+# The concept of OOP In Java Getting Started with ABSTRACTION and ENCAPSULATION
 
-In Java, everything is considered an object and is treated as such; a real-life entity.
+In Java, everything is an object and as such; a real-life entity.
```

Comment:
> In Java, everything is an object and as such; a real-life entity.
> 
> Object-Oriented Programming (OOP) unlike Procedural programming is an approach. Or a programming paradigm where everything is an Object (Real-life entities). OOP tries to model real-life objects.

You first mention the idea of everything being an object in Java and then also in OOP in a way that I think doesn't flow well. Maybe remove that first sentence about Java and add it after talking about how everything is an object in OOP. You would say that since Java is an OOP language, everything is an object. Also, the grammar and formatting in that paragraph are a little off to where the intended meaning is hard to find. I would rephrase this part as so:

> Object-Oriented Programming (OOP) is a programming paradigm (approach) as is procedural programming. Unlike procedural programming, we represent everything as objects (real-life entities). Java, as an object-oriented programming language, was designed to follow this pattern.

Reply:
> OOP tries to model real-life objects.

I don't believe this part is necessary